crates/uv-client/src/base_client.rs

```diff
@@ -136,6 +137,9 @@ impl<'a> BaseClientBuilder<'a> {
 
         // Initialize the base client.
         let client = self.client.clone().unwrap_or_else(|| {
+            // Use aws_lc_rs as the default TLS provider to support more SSL cert algorithms
+            aws_lc_rs::default_provider().install_default().expect("failed to install aws_lc_rs as default TLS provider");
```

Comment:
Can we avoid panicking here? How can this fail? Can we get away with just warning if it fails?